checker/tests/index/SameLenNewArrayWithSameLength.java

```diff
@@ -0,0 +1,7 @@
+import org.checkerframework.checker.index.qual.*;
+
+class SameLenNewArrayWithSameLength {
+    public void m1(int[] a) {
+        int @SameLen("a") [] b = new int[a.length]; // index checker warning
```

Comment:
The `index checker warning` is presumably before your fix. If it is expected, it should be a proper expected output. If not, reformulate to make clear that there used to be a warning.